### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by jiangdengke** [simple-robot/Discussions#164] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

就是类似于一个解析抖音的监听函数，当第一次收到命令解析抖音的过程中，如果再次触发这个监听，知道上次监听的结果，来判断是否进行这次解析

**Comment by ForteScarlet** [comment]
自行借助各种形式的缓存来记录上一次的处理结果即可。例如 Map、redis 等。

**Reply by jiangdengke** [reply]
请问这个
 // 返回 EventResult.truncate 代表阻止后续其他监听函数的执行。
        return EventResult.truncate();
大致是什么意思，或者与返回值是void有什么区别

**Reply by ForteScarlet** [reply, marked as answer]
参考 #62 
字面意思，返回 `EventResult.truncate()` 会截断低优先级的其他监听函数的执行，返回 void 则不会。
